src/events/client/interactionCreate.ts

```diff
@@ -47,6 +57,45 @@ export default function interactionCreateHandler(
         );
     } else if (interaction instanceof Eris.CommandInteraction) {
         if (
+            interaction.data.type ===
+            Eris.Constants.ApplicationCommandTypes.CHAT_INPUT
+        ) {
+            const messageContext = new MessageContext(
+                interaction.channel.id,
+                new KmqMember(
+                    interaction.member.username,
+                    getUserTag(interaction.member),
+                    interaction.member.avatarURL,
+                    interaction.member.id
+                ),
+                interaction.guildID
+            );
+
+            logger.info(
+                `${getDebugLogHeader(interaction)} | ${
+                    interaction.data.name
+                } slash command received`
+            );
+
+            if (interaction.data.name === "groups") {
```

Comment:
We should refactor all interactions in this file to work based off of the interaction name, instead of just the interaction type